### PR TITLE
Feature support lammps data files

### PR DESCRIPTION
**Pull request by JoshuaSBrown**

### CODE REVIEW

File: include/votca/csg/topologyreader.h [hunk 1]
```diff
@@ -31,7 +31,7 @@ class TopologyReader
 {
 public:
     virtual ~TopologyReader() {}
-    /// open a trejectory file
+    /// open a trajectory file
```
Comment (junghans) [inline include/votca/csg/topologyreader.h]: I guess topology file.

File: src/libcsg/modules/io/lammpsdatareader.h [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+        void Close();
+        
+    private:
+
+        std::ifstream fl_;
+        std::string fname_;
+        bool topology_;
+
+				std::map<std::string,std::vector<std::vector<std::string>>> data_;
```
Comment (junghans) [inline src/libcsg/modules/io/lammpsdatareader.h]: Indentation screwed up

File: src/libcsg/modules/io/lammpsdatareader.cc [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+
+				// See if the line is the lammps .data header/info line
+				labelMatched = MatchFieldsTimeStepLabel_(fields,top);
+
+				if(!labelMatched){
+					string err = "Unrecognized line in lammps .data file:\n"+line;
+					throw runtime_error(err);
+				}
+			}
+			getline(fl_, line);
```
Comment (junghans) [inline src/libcsg/modules/io/lammpsdatareader.cc]: I guess you should free all the helper structures here!

**Review by junghans — CHANGES_REQUESTED** [review]
And convert tabs to spaces everywhere!

**Comment by JoshuaSBrown** [comment]
@junghans I think I have addressed your comments but take a look.

**Review by junghans — APPROVED** [review]

**Pull request merged**